discovery_engine/lib/src/api/events/client_events.dart

```diff
@@ -189,6 +189,15 @@ class ClientEvent with _$ClientEvent {
   @Implements<SearchClientEvent>()
   const factory ClientEvent.trendingTopicsRequested() = TrendingTopicsRequested;
 
+  /// Resets the AI state of the engine.
+  ///
+  /// This does clear the history.
```

Comment:
:art: We never expose the concept of history outside the engine so I don't think we should mention it.